### DISCUSSION TITLE
How to customize navbar

### DISCUSSION THREAD

**Discussion by khaldon** [gregsullivan/_tw#107] [General]
It's so difficult to customize navbar how can I do that ? i would like to use preline taiwind navbar in wordpress but I can't

**Comment by gregsullivan** [comment]
Hello—Thanks for posting your question!

Just to give me some context, have you already read the documentation page on [styling HTML from outside the theme](https://underscoretw.com/docs/styling-html-from-outside-the-theme/)? It might provide some useful details.

That aside, where are you running into trouble? Have you done the complete Preline installation and confirmed the JavaScript file is being loaded in your frontend?

Meanwhile, I'm going to convert this to a discussion, and you can provide any further details there.
